README.adoc

```diff
@@ -192,18 +193,18 @@ Create the `InventoryManager` class in the
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[tags=add_retry_fallback]
 ----
 
-The `@Retry` annotation dictates on what conditions we should reattempt to establish a connection
-to the `system` service and how many times it should be retried, in this case three times.
-The `@Fallback` annotation dictates which method to call when the retries are unsuccessful, in this
-case the fallbackForGet method can be used.
+The `@Retry` annotation dictates the conditions that determine if the microservice should reattempt to establish a connection
+to the `system` service and how many times it should retry. In this example, the microservice retries three times.
+The `@Fallback` annotation dictates which method to call when the retries are unsuccessful. In this
+example, you can use the `fallbackForGet` method.
 
-The `fallbackForGet()` method that has been added as the designated fallback method for the original
-get() method, checks to see if the system's properties exist in the inventory. If the properties do 
-not exist, this suggests that the application has not been able to successfully reach the `system`
-service to get the system’s properties previously, thus the method simply prints out a json warning 
+The `fallbackForGet()` method, which is added as the designated fallback method for the original
+`get()` method, checks to see if the system's properties exist in the inventory. If the properties do 
+not exist, the application has likely been unsuccessful in reachig the `system`
```

Comment:
=> How about, "If the properties is null, this means the application did not reach the `system` service successfully to get the system’s properties previously." The way you put it, has a different meaning.

Reply:
Unless we're talking about a single value that is called "properties," we want to use a plural verb ("are").

I'm also not quite sure what "previously" is describing. I think we want it to describe when the system service tried to get the system properties, correct?

What about this version?
"If the properties are null, the application did not reach the system service successfully and previously could not reach the system’s properties."
or
"If the properties are null, the application could not reach the system's properties and did not reach the system service successfully."

Reply:
Actually "properties" is a variable defined in the code. How about, "If the properties variable is null, the system properties entry is not found in the inventory."?